script/vsts/platforms/templates/preparation.yml

```
@@ -0,0 +1,66 @@
+steps:
+
+  # Linux Specific
+  - script: |
+      sudo apt-get update
+      sudo apt-get install -y wget software-properties-common
+      sudo apt-get install -y build-essential ca-certificates xvfb fakeroot git libsecret-1-dev rpm libx11-dev libxkbfile-dev xz-utils xorriso zsync libxss1 libgconf2-4 libgtk-3-0 libasound2 libicu-dev
+      # clang 9 is included in the image
+      sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-9 10
+      sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-9 10
+      sudo update-alternatives --config clang
+      sudo update-alternatives --config clang++
+      clang -v
+    displayName: Install apt dependencies
+    condition: eq(variables['Agent.OS'], 'Linux')
+
+
+  - script: sudo /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16
+    displayName: Start Xvfb
+    condition: eq(variables['Agent.OS'], 'Linux')
+
+  # Common
+  - task: NodeTool@0
+    inputs:
+      versionSpec: 12.4.0
+    displayName: Install Node.js 12.4.0
+
+  - script: npm install --global npm@6.9.0
+    displayName: Update npm
+
+  # Use Azure Syntax to set env variables for all shells and steps
+  - pwsh: |
+      if ($env:AGENT_OS -eq "Windows_NT") {
+        $env:BUILD_ARCH=$env:buildArch
+        echo "##vso[task.setvariable variable=BUILD_ARCH]$env:BUILD_ARCH"
+      }
+    displayName: Setting globally used env variables
+
+  # Windows Specific
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.8'
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - script: |
+      ECHO Installing npm-windows-upgrade
+      npm install --global --production npm-windows-upgrade
+    displayName: Install npm-windows-upgrade
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - script: |
+      ECHO Upgrading npm
+      npm-windows-upgrade --no-spinner --no-prompt --npm-version 6.9.0
+    displayName: Install npm 6.9.0
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - script: |
+      npm install --global --production windows-build-tools@4.0
+    displayName: Install windows build tools
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - script: |
+      cd script\vsts
```

This is just a nitpick
```suggestion
      cd script/vsts
```

---

This only runs in CMD and only on Windows. So this is not needed.

---

Fixed in https://github.com/atom/atom/pull/21221/commits/ec373db827b2ac2811602c69004c24d9089ebf87